docs/ReleaseNotes.txt

```diff
@@ -1,4 +1,3 @@
-======
```

Comment:
I guess this is an inadvertent change. Could you please remove? Thank you.

Reply:
Sure, will do it ... sorry about that, it probably crept in when doing rebase from devel to master.

Reply:
Fixed in the last rebase.